frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/StrategyExecution/RolloutParameter/RolloutParameter.tsx

```diff
@@ -28,17 +28,19 @@ export const RolloutParameter: FC<{
     return (
         <>
             <StrategyEvaluationItem type='Rollout %'>
-                <StrategyEvaluationChip label={`${percentage}%`} /> of your base{' '}
-                {stickiness}
-                <span>
-                    {hasConstraints ? 'who match constraints ' : ' '}
-                    is included.
-                </span>
-                {displayGroupId && parameters?.groupId ? (
-                    <StrategyEvaluationChip
-                        label={`groupId: ${parameters?.groupId}`}
-                    />
-                ) : null}
+                <StrategyEvaluationChip label={`${percentage}%`} />
```

Comment:
Not sure what this is rendered as, but it should probably be a span, I think. 
```suggestion
                <StrategyEvaluationChip component='span' label={`${percentage}%`} />
```